### ISSUE TITLE
fail to converge when apply background model

### ISSUE THREAD

**Issue by blacksino** [autonomousvision/sdfstudio#252]
Hello,I have apply foreground mask on my custom dataset and I would like to disable background model to speed up the training,however,it fails to converge.
Here is a single example of my custom dataset :
<img width="227" alt="image" src="https://github.com/autonomousvision/sdfstudio/assets/44363764/461c3f89-2598-4ff9-9bee-e5bf177a8ef1">

When training with background model(grid),it works fine.
How could I resolve this kind of problem.

**Comment by niujinshuchong** [comment]
Hi, I think it should work if you disable background. Did you use black for the background color?

**Comment by blacksino** [comment]
Yes, I have set background color as black,no idea why it fails when I disable background model.
It seems that the object I want to reconstruct is relatively small compared to the bounding box of the axis-aligned bounding box (AABB) obtained after normalizing the camera pose, resulting in excessive learning pressure on the foreground model. If this is the case, can I further trim the AABB using the sparse point cloud obtained from COLMAP?

**Comment by hummat** [comment]
I think you could use `--scale_factor 2.0` as an argument to the `nerfstudio-data` data parser.
